submissions/peterIyiola/DOM/contact.js

```diff
@@ -0,0 +1,35 @@
+let submitForm = document.getElementById('validateForm'),
```

Comment:
Where are you using the js file in your HTML?

Reply:
Send a screenshot of your result?

Reply:
![image](https://user-images.githubusercontent.com/61589702/98081190-f4fb0d80-1e76-11eb-948d-1d1c8054e46c.png)
![image](https://user-images.githubusercontent.com/61589702/98081215-ffb5a280-1e76-11eb-9d5c-79071849ce58.png)

he js file(contact.js) is attached just before the closing body tag in my HTML